.github/workflows/installer.yaml

```diff
@@ -0,0 +1,88 @@
+name: Installer
+
+on:
+  push:
+    branches:
+      - dev
+    # paths:
+    #   - 'install-photobooth.sh'
+  pull_request:
+    branches:
+      - dev
+    # paths:
+    #   - 'install-photobooth.sh'
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: 'ubuntu-latest'
+    container:
+      image: ubuntu:latest
+      ports:
+        - 80
+      volumes:
+        - ${{ github.workspace }}:/workspace
+      env:
+        TZ: Europe/Berlin
+        DEBIAN_FRONTEND: noninteractive
+    defaults:
+      run:
+        working-directory: /workspace
+
+    steps:
+
+      - name: Add common packages to image
+        run: |
+          apt-get -qq update
+          apt-get -qq install -y build-essential apt-utils wget sudo
+
```

Comment:
```
      - name: Setup node
        uses: actions/setup-node@v3
        with:
          node-version: 18
```

Reply:
From the log
```
deb [signed-by=/etc/apt/keyrings/nodesource.gpg] https://deb.nodesource.com/node_18.x nodistro main
```

I bet the _nodistro_ is causing the issue still installin v12

Reply:
> ```
>       - name: Setup node
>         uses: actions/setup-node@v3
>         with:
>           node-version: 18
> ```

we cannot use the node setup since we need to test the installer

Reply:
> From the log
> 
> ```
> deb [signed-by=/etc/apt/keyrings/nodesource.gpg] https://deb.nodesource.com/node_18.x nodistro main
> ```
> 
> I bet the _nodistro_ is causing the issue still installin v12

currently debugging the update scripts